### PR TITLE
LTG-35: add enter-phone-number screen

### PR DESCRIPTION
**Pull request by dbes-gds**
## What

Add Enter Phone Number screen.

### CODE REVIEW

File: src/app.constants.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ export const PATH_NAMES = {
   COOKIES_POLICY: "/cookies",
   ENTER_EMAIL: "/enter-email",
   ENTER_PASSWORD: "/enter-password",
+  ENTER_PHONE_NUMBER: "/enter-phone-number",
   CREATE_ACCOUNT_CHECK_EMAIL: "/check-email",
   CREATE_ACCOUNT_SET_PASSWORD: "/create-password",
   CREATE_ACCOUNT_ENTER_PHONE_NUMBER: "/enter-phone-number",
```
Comment (sanisc) [inline src/app.constants.ts:11]: double entry here

File: src/controllers/enter-phone-number-controller.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+                return req.t("pages.enterPhoneNumber.phoneNumber.validationError.required", {value});
+            })
+    ];
+}
+
+export function enterPhoneNumberGet(req: Request, res: Response): void {
+    res.render("enter-phone-number.html");
+}
+
+// export function enterPhoneNumberPost(req: Request, res: Response): void {
```
Comment (sanisc) [inline src/controllers/enter-phone-number-controller.ts:19]: lets remove this code if it is not used

**Review by sanisc — CHANGES_REQUESTED** [review]

File: src/app.constants.ts (hunk 1 above)
Comment (dbes-gds) [reply src/app.constants.ts]: removed

File: src/controllers/enter-phone-number-controller.ts (hunk 2 above)
Comment (dbes-gds) [reply src/controllers/enter-phone-number-controller.ts]: deleted

**Review by sanisc — APPROVED** [review]

**Pull request merged**